### ISSUE TITLE
nikkud error in parshat bo

### ISSUE THREAD

**Issue by jclerman** [akivajgordon/tikkun.io#101]
Hi, I think I've found an error in nikkud: In Shmot 10:28 (in the 3rd aliyah of parshat Bo), we should have "al-tosef" but in the chumash text it's now "el-tosef".

![image](https://user-images.githubusercontent.com/19315670/147868847-44339933-7b9e-40a9-9929-71fb6afba9a9.png)

I compared with a couple other sources; all have "al-tosef".

Thanks and congrats on your work on this resource - it's pretty great!

**Comment by akivajgordon** [comment]
@jclerman – thanks for reporting and for your kind words!

The backstory is that the original source text was grabbed from Sefaria, which is using the Leningrad Codex. It appears that Sefaria uses a different source for its website now, and they have an API that I can get enough bits and pieces with to replace the source text.

In fact, I'm working on that as we speak in order to resolve several other errors (trop errors, word errors, and the lack of kamatz katan), and have it mostly working on my local machine. This particular error that you found appears to be fixed with these changes.

In summary, stay tuned for an update that will make the source text way better, though I can't promise it'll be out before we read parshat Bo this year. Thanks again for reporting. Don't be shy about creating more issues as well.

While I have you here, how did you discover this app?

**Comment by jclerman** [comment]
Got it, thanks @akivajgordon!  

I think I found tikkun.io via HebCal's links, e.g. the book icons on this page: https://www.hebcal.com/sedrot/bo.

I actually wanted to install it locally on my laptop for offline use, but got a bit stuck - I may be missing some prerequisites.  It's the first javascript app I've tried to install; I got npm (version 8.1.2) installed, but (after cloning the repo and `cd`ing into it) `npm install` didn't work for me - more detailed instructions might be needed for a javascript noob like me.

**Comment by akivajgordon** [comment]
@jclerman – thanks for sharing!

Regarding the `npm install` error, I'm not sure what exactly didn't work for you (feel free to paste error messages and I'll be happy to take a look). But you actually don't need npm in order to _run_ it locally. This is a static website, and all the files are already prebuilt for GitHub pages, so you just need to start a static web server. If you have python installed, in the root of the repository, you can execute:

```
# for python 3
python -m http.server 8000
```

or

```
# for python 2
python -m SimpleHTTPServer 8000
```

Feel free to choose whatever port you want (8000 in the examples above), and then visit `http://localhost:$PORT` in your browser.

**Comment by akivajgordon** [comment]
Fixed with #102
